app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -80,13 +80,6 @@ class GithubWebhook extends RequestHandler<Body> {
         case 'pull_request':
           await _handlePullRequest(stringRequest);
           break;
-        case 'check_suite':
-          final CheckSuiteEvent checkSuiteEvent = CheckSuiteEvent.fromJson(
-            jsonDecode(stringRequest) as Map<String, dynamic>,
-          );
-          await githubChecksService.handleCheckSuite(
-              checkSuiteEvent, luciBuildService);
-          break;
```

Comment:
With these lines removed, shall this test be removed: https://github.com/flutter/cocoon/blob/master/app_dart/test/request_handlers/github_webhook_test.dart#L1513

Reply:
Also the handleCheckSuite function: https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/service/github_checks_service.dart#L43

Reply:
The test was already removed in this CL. I'd like to keep the method in github_checks_service, the correct workflow should use it but there is a bug in the apis that forced us to use the pull request instead. If the apis get fixed at some point then we can get back to the initial implementation.